### PR TITLE
Display aliases on profile screen

### PR DESCRIPTION
**Pull request by wsarles**
#979

### CODE REVIEW

File: Source/UI/About/AliasCountButton.swift [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        }
+    }
+
+    var body: some View {
+        Button {
+            showingAliases = true
+        } label: {
+            Text("+\(count)")
+                .foregroundLinearGradient(
+                    LinearGradient(
```
Comment (mplorentz) [inline Source/UI/About/AliasCountButton.swift:42]: Maybe you could add this gradient to `LinearGradient+Planetary.swift`, or maybe it's already there?

Comment (mplorentz) [reply Source/UI/About/AliasCountButton.swift:42]: Same in `AliasListView`.

**Review by mplorentz — COMMENTED** [review]
Looking good! I know we didn't have fully specced designs for this, so this is more design feedback than anything else, but:
- I think it would be more useful to have the "COPY ID" button on the `AliasListView` be a button that copies the Alias HTTP link. That way users could send it straight to their friends.
- I see that the designs are showing aliases like an email address with `@`, like `rose@planetary.name`. I find this pretty unhelpful as the point of making an alias is that it is a tappable, pasteable link, like `rose.planetary.name`. @Chardot is there a reason we don't display them this way on the profile screen?

**Comment by mplorentz** [comment]
Oh, I just noticed another bug. It looks like the aliases I registered while running these branch are showing up on all profile pages, not just mine!

**Comment by wsarles** [comment]
> Oh, I just noticed another bug. It looks like the aliases I registered while running these branch are showing up on all profile pages, not just mine!


b13aec5 should fix this.

**Review by mplorentz — APPROVED** [review]
Thanks for making those changes! I pushed up one more just to change the "COPY ID" button to just say "COPY" now that it is copying the whole alias. Approved!

**Pull request merged**